src/VscPreviewExtension/.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules/
-out/
\ No newline at end of file
+out/
+DfmParse/
+!.vscode/
```

Comment:
so I think it should be checked in? is it?

---
In reply to: [80604219](https://github.com/dotnet/docfx/pull/736#discussion_r80604219) [](ancestors = 80604219)